en-US/Punctuation.xml

```diff
@@ -185,32 +223,43 @@
 
 			</listitem>
 
-		</itemizedlist>
-		 <para>
-			Use a semicolon to separate items in a series if the items contain commas.
+		</itemizedlist>		
+
+		<para>
+			In the following example, a colon is used for a step that introduces or leads into a command:	
 		</para>
-		 <itemizedlist>
+		<itemizedlist>
 			<listitem>
 				<para>
-					Everyday I have coffee, toast, and fruit for breakfast; a salad for lunch; and a peanut butter sandwich, cookies, ice cream, and chocolate cake for dinner.
+					2. Use the <command>ansible-navigator run</command> command to run the <filename>allfacts.yml</filename> playbook:
 				</para>
+<screen>[student@workstation manage-facts]$ ansible-navigator run allfacts.yml
+PLAY [Gather and display facts for managed nodes] ****************************</screen>
 
 			</listitem>
 
-		</itemizedlist>
-		 <para>
-			Use a semicolon before a conjunctive adverb, such as <emphasis>however, therefore, otherwise, namely, for example</emphasis>, and so on.
-		</para>
+		</itemizedlist>	
 
-		 <itemizedlist>
+		<para>
+			In the following example, a period is used at the end of each of several sentences that introduce a code example:	
```

Comment:
This feels like a nitpicky comment (sorry!), but I wonder if we can avoid the stack of prepositional phrases here (to help with clarity). Maybe: "In the following example, several sentences introduce a code example, and each one ends in a period:"